src/main/java/uk/gov/hmcts/ccd/data/user/DefaultUserRepository.java

```diff
@@ -128,24 +131,26 @@ public IdamUser getUser(String userToken) {
     @Override
     public UserDefault getUserDefaultSettings(final String userId) {
         try {
-            final HttpEntity requestEntity = new HttpEntity(securityUtils.authorizationHeaders());
-            final Map<String, String> queryParams = new HashMap<>();
-            queryParams.put("uid", ApplicationParams.encode(userId.toLowerCase()));
+            final HttpHeaders headers = securityUtils.authorizationHeaders();
+            headers.set(EMAIL_HEADER, userId.toLowerCase());
```

Comment:
email header is not meant to be an authorisation header. the code reads misleadingly.

Reply:
renamed the variable 'headers' to 'requestHeaders'.

Reply:
It looks like requestEntity.getHeaders() returns a readonly list of headers and we can't add any more headers to it.